src/views/Home/components/Balances.tsx

```diff
@@ -68,10 +68,10 @@ const Balances: React.FC = () => {
       const supply = await getSushiSupply(honey)
       setTotalSupply(supply)
     }
-    if (honey) {
+    if (honey && account && getSushiAddress(honey)) {
```

Comment:
`getSushiAddress` Already checks if honey is nullable 
```suggestion
    if (account && getSushiAddress(honey)) {
```